app-utils/date-utils.js

```diff
@@ -0,0 +1,25 @@
+const getPreviousWeekDates = () => {
+  const currentDate = new Date();
+
+  const mondayOfPreviousWeek = new Date(currentDate);
+  mondayOfPreviousWeek.setDate(currentDate.getDate() - currentDate.getDay() - 6);
```

Comment:
it is a bad practice to use some abstract integer values - the better way is to store them as constants (e.g. MONDAY_OFFSET)